docs/root/intro/version_history.rst

```diff
@@ -1,6 +1,30 @@
 Version history
 ---------------
 
+0.4.0 (March 23, 2021)
+======================
+
+This is a large release. Moving forward the team will aim to release smaller version updates.
+The following is a very high-level overview of the larger changes going into this release.
+
+Richer Platform-level Feature Set:
+- Adds pluggable logging capabilities via :ref:`setLogger <api_starting_envoy>`
+- Adds :ref:`platform APIs <api_stats>` for emitting time-series data
+- Adds platform Filters
```

Comment:
@goaway could we add documentation for platform filters? Or at least a landing page that summarizes, and we can add full docs in 0.4.1?